sendinblue.php

```diff
@@ -32,16 +32,19 @@
     include_once (_PS_CLASS_DIR_ . '/../classes/Customer.php');
```

Comment:
include (dirname(__FILE__) . '/classes/Customer.php');
this was the good utilisation to catch your file you want to include. Please use dirname(__FILE__).